packages/@lightningjs/ui-components/src/components/TextBox/TextBox.d.ts

```diff
@@ -18,27 +18,107 @@
 
 import lng from '@lightningjs/core';
 import Base from '../Base';
-import type { StylePartial } from '../../types/lui';
-import type { TextContent } from '../InlineContent/InlineContent';
+import { StylePartial } from '../../types/lui';
+import Marquee from '../Marquee';
+import InlineContent from '../InlineContent';
 
-export type TextBoxStyle = {
+// TODO: Properly export TextContent from root level of ui-components so we don't have to grab it from InlineContent
+type ContentBase = {
+  title?: string;
+  announce?: string;
+  style?: string | Record<string, unknown>;
```

Comment:
is this object expected to be the same as the `textStyle` in the TextBoxStyle type below? we could use that same lng.textures.TextTexture.Settings here as well if so